scripts/generate-report.py

```
@@ -87,7 +87,20 @@
 # SQL for the 3 data pulls (separated by marker in SSH output)
 MARKER = "---REPORT-MARKER---"
 
-POSTS_SQL = """
+def _env_bool(name: str, default: bool = False) -> bool:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)